src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -308,14 +308,29 @@ export class RunParametersAction extends TooltipFromLabelAction {
 		const editor = this._notebookService.findNotebookEditor(context);
 		// Set defaultParameters to the parameter values in parameter cell
 		let defaultParameters = new Map<string, string>();
-		editor.cells.forEach(cell => {
+		let emptyParameterCell: boolean = false;
```

Comment:
You don't need to define this up here anymore - just do

`const emptyParameterCell = cellSource.every(s => /^\s*$/.test(s));` below